frontend/tsconfig.json

```diff
@@ -17,6 +17,7 @@
       "es2019",
       "dom",
       "es2020.string",
+      "es2021.string",
```

Comment:
From my understanding, `es2020.string` allows to use all the new methods released for `es2020` (including `matchAll` which is used in v4)
`es2021.string`, instead, allows to use all the methods released for `es2021` (including `replaceAll`).

A more recent version (like `es2021`), however, does not automatically polyfills the methods from the previous `es` versions. What happens is that each version only polyfills the new methods introduces for that `es` version.

The above idea (that a given `es` version polyfills also older versions) would only be valid if, instead of adding these values to the `lib` property, we were to upgrade the `target` property.

> Following is the error if I remove the `es2020.string` from the `lib` property
![image](https://user-images.githubusercontent.com/43547703/216011639-4622a249-9e81-4b6f-9c97-924496e3558f.png)